ReactAndroid/src/main/java/com/facebook/react/modules/websocket/WebSocketModule.java

```diff
@@ -187,4 +200,30 @@ private void notifyWebSocketFailed(int id, String message) {
     params.putString("message", message);
     sendEvent("websocketFailed", params);
   }
+
+  private static String setDefaultOrigin(String uri) {
```

Comment:
I cannot tell what this does based on the name. Can you add short Javadoc please?